crates/error/CHANGELOG.md

```diff
@@ -2,4 +2,8 @@
 
 ## 0.1.0
 
+### Major
+
+- Add an OutOfBounds error
```

Comment:
This is just a minor change because the enum is non-exhaustive.

Also you need to create a `0.1.1-git` section at the top and modify the `Cargo.toml` appropriately. You'll also need to `git grep '^wasefire-error = '` and fix all versions there. It probably works to do `sed -i '/^wasefire-error = /{s/0\.1\.0/0.1.1-git/}' $(git grep '^wasefire-error = ')`. A lot of Cargo.lock file will change so you'll have to run `./scripts/ci.sh` and commit all changes.